modules/troubleshooting-network-observability-loki-tenant-rate-limit.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+
+// * networking/network_observability/troubleshooting-network-observability.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-troubleshooting-loki-tenant-rate-limit_{context}"]
+= LokiStack rate limit errors
+A rate-limit placed on the Loki tenant can result in potential temporary loss of data and a 429 error: 'Per stream rate limit exceeded (limit:xMB/sec) while attempting to ingest for stream'. You might consider having an alert set to notify you of this error. For more information, see _Creating Loki rate limit alerts for the NetObserv dashboard_ in the Additional Resources of this section.
+
+You can update the LokiStack CRD with the `perStreamRateLimit` and `perStreamRateLimitBurst` specifications, as shown in the following the procedure.  
+
+.Procedure
+. Navigate to *Operators* -> *Installed Operators*, viewing *All projects* from the *Project* dropdown.
+. Look for *Loki Operator*. In the details, under *Provided APIs*, select *LokiStack*.
+. Edit the YAML in the *YAML view* to add the `perStreamRateLimit` and `perStreamRateLimitBurst` specifications:
+[source, yaml]
+----
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+metadata:
+  name: loki
+  namespace: netobserv
+spec:
+  limits:
+    global:
+      ingestion:
+        ingestionBurstSize: 100
+        ingestionRate: 500
+        maxGlobalStreamsPerTenant: 100000
+        perStreamRateLimit: 6
+        perStreamRateLimitBurst: 30
+  tenants:
+    mode: openshift-network
+  managementState: Managed
+----
+. Click *Save*.
+
+.Verification
+Once you update the `perStreamRateLimit` and `perStreamRateLimitBurst` specifications, the pods in your cluster restart and ideally no flows are dropped due to rate-limit errors.
```

Comment:
I would clarify this to mention that the 429 error should no longer occur - right now it reads as if no flows will be dropped while the pods are restarting, when actually that will likely be the case since the LokiStack will be down for a few minutes.